crates/editor/src/editor.rs

```diff
@@ -4615,6 +4616,7 @@ impl Editor {
             .size(ui::ButtonSize::None)
             .icon_color(Color::Muted)
             .selected(is_active)
+            .tooltip(|cx| Tooltip::text("Run", cx))
```

Comment:
<img width="473" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/19876735-abba-4713-b31b-af686890673c">

Not sure how useful it is at this form: the icon and the pointing hand caret seems to pass this message without extra verbosity?

Also, for cases when both code actions and running are combined, it starts to look even more odd:
<img width="702" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/7a6b0091-4291-4c4f-9915-26ea2e9fd88c">